Lib/asyncio/base_events.py

```diff
@@ -1570,6 +1571,10 @@ def _log_subprocess(self, msg, stdin, stdout, stderr):
             raise ValueError("shell must be True")
         if bufsize != 0:
             raise ValueError("bufsize must be 0")
+        if encoding or errors or text:
```

Comment:
Please split this `if` into three separate checks. It makes error messages more specific.